docs/modules/migrate/pages/data-migration-tool.adoc

```diff
@@ -17,18 +17,26 @@ When using the DMT, bear the following in mind:
 ** The source cluster must be running and populated with required data
 ** The application must be using the source cluster successfully before migration
 ** Before starting the migration, remember to shutdown any applications that are using the source cluster
-** If migrating Replicated map metadata, you must add the custom JAR to your class path and restart the cluster. If this is not done, the Replicated map metadata is not migrated. For further information on adding the custom JAR to your classpath, refer to the https://docs.oracle.com/javase/6/docs/technotes/tools/windows/classpath.html[Oracle documentation,window=_blank].
+** For certain versions, you must add the provided extension JAR to your class path and restart the cluster if you want to migrate Replicated map metadata and/or CP data structures. For further information on the affected versions and use of the extension JAR, see the xref:migrate:dmt-source-cluster-extension.adoc[] topic
++
+If needed, but not done, Replicated map metadata and/or CP data structure are not migrated. For further information on adding the extension JAR to your classpath, refer to the https://docs.oracle.com/javase/6/docs/technotes/tools/windows/classpath.html[Oracle documentation,window=_blank].
```

Comment:
almost true. 

If the user does not set the "migrateReplicatedMapMetadata" option to false explicitly, the migration cluster will try to migrate replicated map metadata by default and there will be an error & migration fail about the missing API.

The CP migration is explicit by the user, we only migrate CP migrations if there are entries in the CP files in  migration_config/data folder, used to select which CP structures to migrate. If they select CP data structures but the API is absent on the source side, we'll again fail the migration.

Reply:
I have amended and expanded this description in the latest update.